base.js

```diff
@@ -213,6 +213,8 @@ const styleRules = {
 
     "operator-linebreak": [2, "after"],
 
+    "semi": 2,
```

Comment:
Hmmmmm, is it possible to use "semi" and "babel-semi" rule at the same time? For example if we have a project that doesn't use a babel then it would be nice to still require semicolons inside of JS code.